### ISSUE TITLE
[Feature Request] Fully Connected Network for Regression

### ISSUE THREAD

**Issue by acxz** [PyTorchLightning/pytorch-lightning-bolts#393]
## 🚀 Feature

Adding a fully connected (FC) multilayer perceptron (MLP) Network for Regression tasks.

### Motivation

For simple regression tasks FC networks are commonly used. I saw that we have a LinearRegression model, but it would also be nice to have a full blown MLP network that people can just plug and chug for there supervised learning tasks.

### Pitch

under `pl_bolts/models/regression/` add `mlp_regression.py`, with the ability to specify the hidden layers and the hidden dims.

### Alternatives

Usually people just roll their own solution for regression so it be a bit too specific to have an MLP network inside the `pl_bolts` repo, but I'm sure people looking for vector input to vector output regression problems would appreciate and solution that can be immediately used. Haven't seen an out of the box MLP solution elsewhere (maybe b/c it is kinda easy to implement on your own).

**Comment by akihironitta** [comment]
@acxz Thank you for your suggestion! It'd be nice to have a simple MLP regressor in Bolts. Once we have positive comments from other core members, would you be interested in submitting a PR?

**Comment by acxz** [comment]
Yeah I have been spinning one up today and can PR it once I finish/clean it up.

**Comment by djbiega** [comment]
Hi, I'd be interested in adding this in as an easy first issue if this is still desired. Could I get this assigned to me?
